torch/_inductor/utils.py

```diff
@@ -2776,6 +2776,24 @@ def triton_version_uses_attrs_dict() -> bool:
     return get_triton_attrs_descriptor_version() == TritonAttrsDescriptorVersion.V4_DICT
 
 
+def is_cudagraph_unsafe_custom_op(node: Operation) -> bool:
+    """
+    Returns True if the node is a custom op that is not cudagraphable.
+    """
```

Comment:
nit: built-in operators may have the cudagraph_unsafe tag too. So the function should probably be named `is_cudagraph_unsafe_op`, and the description should say "usually only custom ops will have these"